README.md

```diff
@@ -37,36 +28,39 @@ go get -u gopkg.in/src-d/go-git.v4/...
 Examples
 --------
 
-Retrieving the commits for a given repository:
+Cloning a repository and printing the history of HEAD, just like `git log` does
+
+> You can find this [example](examples/log/main.go) and many other at the [examples](examples) folder
 
 ```go
-r, err := git.NewRepository("https://github.com/src-d/go-git", nil)
-if err != nil {
-	panic(err)
-}
+// We instance an in-memory git repository
+r := git.NewMemoryRepository()
 
-if err := r.PullDefault(); err != nil {
-	panic(err)
-}
+// Clone the given repository, creating the remote, the local branches
+// and fetching the objects, exactly as:
+Info("git clone https://github.com/src-d/go-siva")
 
-iter, err := r.Commits()
-if err != nil {
-	panic(err)
-}
-defer iter.Close()
+err := r.Clone(&git.CloneOptions{URL: "https://github.com/src-d/go-siva"})
+CheckIfError(err)
+
+// Getting the HEAD history from HEAD, just like does:
```

Comment:
In the previous comment you used the imperative voice to define the action the code is doing.